torch/_inductor/select_algorithm.py

```diff
@@ -120,7 +120,10 @@ def jit_line(self):
         }
         triton_meta["configs"] = [config_of(signature)]
 
-        inductor_meta = {"kernel_name": str(Placeholder.DESCRIPTIVE_NAME)}
+        inductor_meta = {
+            "kernel_name": str(Placeholder.DESCRIPTIVE_NAME),
+            "origin_ops": str(Placeholder.ORIGIN_INFO),
```

Comment:
Can we include `origin_ops` in the `inductor_meta` only if `profiler_mark_wrapper_call` is enabled ?